sys/linux/nbd.txt

```diff
@@ -8,15 +8,16 @@ include <uapi/linux/nbd.h>
 include <uapi/linux/nbd-netlink.h>
 
 resource fd_nbd[fd_block]
-resource sock_nbd_client[sock]
-resource sock_nbd_server[sock]
+resource sock_nbd[sock_in]
 resource genl_nbd_family_id[int16]
 
+socket$inet_nbd(domain const[AF_INET], type const[SOCK_STREAM], proto const[0]) sock_nbd
```

Comment:
We actually want a connected socket pair here. Please just change the socketpair call to use AF_UNIX, but leave the rest (e.g. separate resources for client/server side).

Reply:
Should be fixed in https://github.com/google/syzkaller/pull/846/commits/41133aabf60f50664118852b68f27e5674f89d6c.